CalendarKit-Swift/SwiftCal/ICSEventParser.swift

```diff
@@ -37,13 +39,13 @@ class ICSEventParser: NSObject {
     }
 
     static func event(from icsString: String) -> CalendarEvent? {
-        
-        guard let timezone = timezone(from: icsString) else { return nil }
-        
+
+        let timeZone = timezone(from: icsString)
```

Comment:
probably worth a comment in the pr so the maintainer knows that the timezone is not always preset